__tests__/html2/fluentTheme/defaultFeedback.activity.html

```diff
@@ -1,127 +1,164 @@
 
-        await host.snapshot('local');
+      await host.snapshot('local');
 
-        // Click like button
-        await host.sendKeys('ENTER');
+      // Click like button
+      await host.sendKeys('ENTER');
 
-        await pageConditions.became(
-          'feedback form is open',
-          () => document.activeElement === pageElements.byTestId('feedback sendbox'),
-          1000
-        );
+      await pageConditions.became(
+        'feedback form is open',
+        () => document.activeElement === pageElements.byTestId('feedback sendbox'),
+        1000
+      );
 
-        // Go to cancel button
-        await host.sendTab(2);
-        await host.sendKeys('ENTER');
+      // Go to cancel button
+      await host.sendTab(2);
+      await host.sendKeys('ENTER');
 
-        await host.snapshot('local');
+      await host.snapshot('local');
 
-        pageElements.byTestId('send box text area').focus();
-        await host.sendShiftTab(2);
+      pageElements.byTestId('send box text area').focus();
+      await host.sendShiftTab(2);
 
-        await host.sendKeys('ENTER');
-        // Send dislike
+      await host.sendKeys('ENTER');
+      // Send dislike
+      await host.sendTab(1);
+      await host.sendKeys('ENTER');
+
+      await pageConditions.became(
+        'feedback form is open',
+        () => document.activeElement === pageElements.byTestId('feedback sendbox'),
+        1000
+      );
+
+      await host.sendKeys('Test feedback');
+
+      await host.snapshot('local');
+
+      const { activity } = await directLine.actPostActivity(async () => {
         await host.sendTab(1);
         await host.sendKeys('ENTER');
+      });
 
-        await pageConditions.became(
-          'feedback form is open',
-          () => document.activeElement === pageElements.byTestId('feedback sendbox'),
-          1000
-        );
-
-        await host.sendKeys('Test feedback');
-
-        await host.snapshot('local');
-
-        const { activity } = await directLine.actPostActivity(async () => {
-          await host.sendTab(1);
-          await host.sendKeys('ENTER');
-        });
-
-        expect(activity).toEqual(
-          expect.objectContaining({
-            type: 'invoke',
-            name: 'message/submitAction',
-            value: {
-              actionName: 'feedback',
-              actionValue: {
-                reaction: 'dislike',
-                feedback: {
-                  feedbackText: 'Test feedback'
-                }
+      expect(activity).toEqual(
+        expect.objectContaining({
+          type: 'invoke',
+          name: 'message/submitAction',
+          value: {
+            actionName: 'feedback',
+            actionValue: {
+              reaction: 'dislike',
+              feedback: {
+                feedbackText: 'Test feedback'
               }
             }
-          })
-        );
+          }
+        })
+      );
+
+
+      await directLine.emulateIncomingActivity({
```

Comment:
Can we separate this into another test? We don't need to test input (i.e. `sendKeys`), just layout test is fine.